tinygrad/ops.py

```diff
@@ -124,8 +124,7 @@ def optimize_local_size(self, bufs) -> List[int]:
   def lower(self, bufs) -> List[RawBuffer]: return [x.raw() for i,x in enumerate(bufs) if x is not None and i not in self.bufs_to_delete]
   def __call__(self, bufs):
     if getenv("OPTLOCAL") and self.global_size is not None and self.local_size is None: self.local_size = self.optimize_local_size(bufs)
-    et = self.clprg(self.global_size, self.local_size, *bufs, wait=DEBUG>=2)
-    if et is not None: GlobalCounters.time_sum_s += et
+    if et := self.clprg(self.global_size, self.local_size, *bufs, wait=DEBUG>=2): GlobalCounters.time_sum_s += et
```

Comment:
Neat!

Reply:
The scope is not limited to the if block, so there are no issues with it.
```python
>>> a = 2
>>> if a := 1 :
...     print(a)
...
1
>>> a
1


>>> if b := 2:
...     print(b)
...
2
>>> b
2

```